api/src/main/java/org/pmiops/workbench/reporting/insertion/UserColumnValueExtractor.java

```diff
@@ -72,7 +72,20 @@
   INSTITUTIONAL_ROLE_ENUM(
       "institutional_role_enum", u -> enumToString(u.getInstitutionalRoleEnum())),
   INSTITUTIONAL_ROLE_OTHER_TEXT(
-      "institutional_role_other_text", ReportingUser::getInstitutionalRoleOtherText);
+      "institutional_role_other_text", ReportingUser::getInstitutionalRoleOtherText),
+  DISABILITY("disability", u -> enumToString(u.getDisability())),
+  DEGREES("degrees", ReportingUser::getDegrees),
+  ETHNICITY("ethnicity", u -> enumToString(u.getEthnicity())),
+  GENDER_IDENTITY("gender_identity", ReportingUser::getGenderIdentity),
+  HIGHEST_EDUCATION("highest_education", u -> enumToString(u.getHighestEducation())),
```

Comment:
nit: I would probably just go with `education` here to match our database. They should be able to easily adapt their reporting queries accordingly.

Reply:
The UI says highest education. So highest_education is closer to what PDR need?